### ISSUE TITLE
django chat response aren't converting into json objects

### ISSUE THREAD

**Issue by vkosuri** [gunthercox/ChatterBot#497]
Captured Traceback of thsi errror

``` Bash
Traceback:  

File "/app/.heroku/python/lib/python2.7/site-packages/django/core/handlers/exception.py" in inner
  39.             response = get_response(request)

File "/app/.heroku/python/lib/python2.7/site-packages/django/core/handlers/base.py" in _legacy_get_response
  249.             response = self._get_response(request)

File "/app/.heroku/python/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/app/.heroku/python/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/app/.heroku/python/lib/python2.7/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/app/.heroku/python/lib/python2.7/site-packages/django/views/generic/base.py" in dispatch
  88.         return handler(request, *args, **kwargs)

File "/app/.heroku/python/lib/python2.7/site-packages/chatterbot/ext/django_chatterbot/views.py" in post
  67.         response_data = self.chatterbot.get_response(input_data, chat_session.id_string)

File "/app/.heroku/python/lib/python2.7/site-packages/chatterbot/chatterbot.py" in get_response
  98.         statement, response, confidence = self.generate_response(input_statement, session_id)

File "/app/.heroku/python/lib/python2.7/site-packages/chatterbot/chatterbot.py" in generate_response
  122.         confidence, response = self.logic.process(input_statement)

File "/app/.heroku/python/lib/python2.7/site-packages/chatterbot/logic/multi_adapter.py" in process
  37.                 confidence, output = adapter.process(statement)

File "/app/.heroku/python/lib/python2.7/site-packages/chatterbot/logic/best_match.py" in process
  59.         confidence, closest_match = self.get(input_statement)

File "/app/.heroku/python/lib/python2.7/site-packages/chatterbot/logic/best_match.py" in get
  23.         statement_list = self.chatbot.storage.get_response_statements()

File "/app/.heroku/python/lib/python2.7/site-packages/chatterbot/storage/storage_adapter.py" in get_response_statements
  88.         statement_list = self.filter()

File "/app/.heroku/python/lib/python2.7/site-packages/chatterbot/storage/django_storage.py" in filter
  80.             results.append(self.model_to_object(statement_object))

File "/app/.heroku/python/lib/python2.7/site-packages/chatterbot/storage/django_storage.py" in model_to_object
  27.             extra_data=json.loads(statement_model.extra_data, encoding='utf8')

File "/app/.heroku/python/lib/python2.7/json/__init__.py" in loads
  352.     return cls(encoding=encoding, **kw).decode(s)

File "/app/.heroku/python/lib/python2.7/json/decoder.py" in decode
  364.         obj, end = self.raw_decode(s, idx=_w(s, 0).end())

File "/app/.heroku/python/lib/python2.7/json/decoder.py" in raw_decode
  382.             raise ValueError("No JSON object could be decoded")

Exception Type: ValueError at /api/chatterbot/
Exception Value: No JSON object could be decoded
Request information:
USER: AnonymousUser
```

**Comment by vkosuri** [comment]
And also it was failed due to unable to convert Unicode data into string

**Comment by gunthercox** [comment]
Closed by #498
